tox.ini

```diff
@@ -4,7 +4,8 @@ envlist=py27,py36,flake8,doclint
 [testenv]
 deps=
     pytest
-commands=pytest -vv
+passenv = *
+commands=pytest -vv -rfsx --tb=short
```

Comment:
Could you explain why you made these changes please? (Not saying they're bad, just want to understand :))

Reply:
I used `passenv = *` so that we can run integration test manually by the command `TRAVIS_EVENT_TYPE='cron' tox`
The `pytest -rfsx --tb=short` is used to report (-r) the summary information about (f)ailed, (s)kipped và (x)fail test, and shorten the traceback.